### ISSUE TITLE
PINT is not reading full parks clock correction file. 

### ISSUE THREAD

**Issue by luojing1211** [nanograv/PINT#319]
Hi all, 
I just realized that when PINT reading pks clock correction file. Some data points are missing from the file. 
```
>>> p = clock_file.TempoClockFile.read('../pint/datafiles/time_pks.dat', site='7')
>>> p1 = clock_file.TempoClockFile('../pint/datafiles/time.dat', obscode='7')
>>> len(p.clock)
5818
>>> len(p1.clock)
5799
```
Does this happens on your computer?

**Comment by luojing1211** [comment]
I found the problem. 
In time_pks.dat the skipped line is like: 
55180.36       0.000      -16.878 7
Which has one more digit compare to the normal line:
55179.51       0.000      -2.022 7
The site code "7" was not parsed correctly. 
@demorest  Do you have time to fix it? Otherwise I will fix it.

**Comment by demorest** [comment]
hi Jing, the tempo clock file format specifies that the site code should be in column 35; see http://tempo.sourceforge.net/ref_man_sections/clock.txt.  So the clock file, rather than the parsing code, should be fixed.  This file would presumably be read incorrectly by tempo as well.

**Comment by demorest** [comment]
PS, all you need to do is delete one of the spaces between the "0.000" and the "-16.878" in your example above to get everything to line up correctly again.

**Comment by luojing1211** [comment]
This issue is fixed by Pull Request #321 Closing.
